src/main/java/frc/robot/constants/Constants2023.java

```diff
@@ -32,6 +32,9 @@ public class Constants2023 extends GameConstants {
     public static final double SHOOTER_MOTOR_LOW_SPEED = 0.6;
     public static final double SHOOTER_MOTOR_HIGH_SPEED = 1;
 
+    public static final double SHOOTER_MOTOR_RIGHT_AMP_SPEED = 0.25;
```

Comment:
These constants are good examples of values that can go into GameConstants so you don't have to repeat them in every constants file.